File: Maximum Ascending Subarray Sum.go
func maxAscendingSum(nums []int) int {
    n := len(nums)
    maxx := nums[0]
    summ := nums[0]
    for i := 1; i < n; i++ {
        if nums[i] > nums[i-1] {
            summ += nums[i]
        } else {
            summ = nums[i]
        }
        maxx = max(maxx, summ)
    }
    return maxx
}

File: Maximum Ascending Subarray Sum.py
class Solution:
    def maxAscendingSum(self, nums: List[int]) -> int:
        maxx = summ = nums[0]
        n = len(nums)
        for i in range(1, n):
            if nums[i] > nums[i-1]:
                summ += nums[i]
            else:
                summ = nums[i]
            maxx = max(maxx, summ)
        return maxx
